packages/custom-functions-metadata-plugin/package.json

```diff
@@ -18,10 +18,13 @@
     "url": "https://github.com/OfficeDev/Office-Addin-Scripts"
   },
   "scripts": {
-    "tsc": "tsc -p tsconfig.json",
-    "tsc-watch": "tsc -p tsconfig.json -w",
     "build": "rimraf lib && concurrently \"npm run tsc\"",
-    "test": "echo \"No tests.\""
+    "lint": "office-addin-lint check",
+    "lint:fix": "office-addin-lint fix",
+    "prettier": "office-addin-lint prettier",
+    "test": "echo \"No tests.\"",
+    "tsc": "tsc -p tsconfig.json",
```

Comment:
I don't think you should add "tsc" since there is already "build". You should rename "tsc-watch" to just "watch".

Reply:
I wasn't adding it . . . I just reorder the commands to be in alpha order.  That said I could go back and remove them if they are unnecessary.

Reply:
I looked add if I understand correctly the build command is actually calling the tsc command.  I'm assuming this needs to be collapsed into the build command (which appears to be the case in the file you said looked good).

Reply:
You shouldn't need to add `-p tsconfig.json` since typescript will use the tsconfig.json file by default. "npm run build" should already be sufficient for building.